api/envoy/extensions/dynamic_modules/v3/dynamic_modules.proto

```diff
@@ -34,10 +33,12 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 // same version of the ABI.
 //
 // Currently, the implementation is work in progress and not usable.
-// [#next-free-field: 8]
 message DynamicModuleConfig {
-  // The location of the object file. Currently, only the local path is supported.
-  config.core.v3.DataSource object_file = 7;
+  // The name of the dynamic module. Envoy searches for the shared object with this name in the module search path.
```

Comment:
Envoy is not the only xDS client, so please make the description less Envoy-centric.  I think we should say something like "The client is expected to have some configuration indicating where to search for the module.  In Envoy, we will (...describe Envoy-specific behavior here...)".

Reply:
Thanks! addressed in https://github.com/envoyproxy/envoy/pull/36448/commits/f9f42c5dd065773f481886bedabb63bec4a4decd